include/abb_librws/rws_poco_client.h

```diff
@@ -246,16 +247,16 @@ class POCOClient
    *
    * \param ip_address for the remote server's IP address.
    * \param port for the remote server's port.
-   * \param user for the remote server's authentication (assumed to be Digest).
-   * \param password for the remote server's authentication (assumed to be Digest).
+   * \param username for the username to the remote server's authentication process.
+   * \param password for the password to the remote server's authentication process.
```

Comment:
Should `for` be removed here?

Reply:
Same comment for all other places that `for` is used in this way.

Reply:
I have thought about improving the parameter descriptions, but I haven't had time yet. I have opened an issue to improve this, see https://github.com/ros-industrial/abb_librws/issues/26